src/Umbraco.Web.UI.Client/src/views/common/infiniteeditors/mediapicker/mediapicker.controller.js

```diff
@@ -264,18 +264,24 @@ angular.module("umbraco")
                                 function (f) {
                                     return f.path.indexOf($scope.startNodeId) !== -1;
                                 });
+							folder.path = $scope.path[0].path;
```

Comment:
Minor point, but is the tabbing slightly out on this method?

Reply:
Dropped this one. But hopefully the fix can still be merged in?